dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry1.1.ee-8.0.feature

```diff
@@ -0,0 +1,21 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpTelemetry1.1.ee-8.0
+singleton=true
+-features=\
+  com.ibm.websphere.appserver.mpRestClient-1.1, \
```

Comment:
You are using mpRestClient public feature here, but in the original mpTelemetry-1.1 it was only use the API feature.  If you use the API feature, then the EE 7 (i.e. mp-1.2 and 1.3) will be easier to do, but will probably still need some work.